terraform/groups/lambda/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,11 @@
+# ------------------------------------------------------------------------------
+# Environment
+# ------------------------------------------------------------------------------
+
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+cron_schedule = "cron(0 23 * * ? *)"
```

Comment:
Will this run daily ? I mean is there a need for this to run daily ?

Reply:
This is now set to run every 6 months, thanks for pointing it out!